tutor/env.py

```diff
@@ -7,13 +7,15 @@
 from copy import deepcopy
 
 import jinja2
-import pkg_resources
+from importlib_resources import files, as_file
```

Comment:
Let's keep the import explicit: `import importlib_resources`. Same in other files.

Reply:
Done!